CI flaky since #1777

CI for PRs + Ci for main are flaky.

From CI logs (in `:nessie-quarkus`):
```
Error:  Errors: 
Error:    TestBasicAuthentication.testValidAdminCredentials:43 » NessieInternalServer In...
Error:    TestBasicAuthentication.testValidCredentials:36 » NessieInternalServer Interna...
```

The preceding errors point to some internal error, probably related to Quarkus' test-profile w/ OIDC issue.

This CI error:
```
2021-08-30T16:04:31.6387453Z [ERROR] org.projectnessie.server.TestBasicAuthentication.testValidAdminCredentials  Time elapsed: 0.825 s  <<< ERROR!
2021-08-30T16:04:31.6389962Z org.projectnessie.client.rest.NessieInternalServerException: 
2021-08-30T16:04:31.6391279Z Internal Server Error (HTTP/500): Internal Server Error
2021-08-30T16:04:31.6393422Z com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "details" (class org.projectnessie.error.NessieError), not marked as ignorable (4 known properties: "serverStackTrace", "status", "reason", "message"])
```
Is a side effect of #1838

--- a/servers/quarkus-server/src/main/java/org/projectnessie/server/authn/NessieHttpAuthenticator.java
+++ b/servers/quarkus-server/src/main/java/org/projectnessie/server/authn/NessieHttpAuthenticator.java
@@ -15,7 +15,6 @@
  */
 package org.projectnessie.server.authn;
 
-import io.quarkus.runtime.Startup;
 import io.quarkus.security.AuthenticationFailedException;
 import io.quarkus.security.identity.IdentityProvider;
 import io.quarkus.security.identity.IdentityProviderManager;
@@ -46,7 +45,6 @@ import org.projectnessie.server.config.QuarkusNessieAuthenticationConfig;
 @Alternative // @Alternative + @Priority ensure the original HttpAuthenticator bean is not used
 @Priority(1)
 @ApplicationScoped
-@Startup // Initialize and load config on the main thread rather than on event loop threads
 public class NessieHttpAuthenticator extends HttpAuthenticator {
 
   @Inject IdentityProviderManager identityProvider;

